doc/en/user/source/rest/stores.rst

```diff
@@ -339,4 +339,64 @@ Given the following content saved as :file:`annotations.xml`:
 
    201 Created
 
-A new and empty table named "annotations" in the "nyc" database will be created as well.
\ No newline at end of file
+A new and empty table named "annotations" in the "nyc" database will be created as well.
+
+
+Adding an external WMTS Store
+-----------------------------
+
+**Create a new WMTS store "Basemap-Nat-Geo-Datastore"**
+
+*Request*
+
+.. admonition:: curl
+
+   ::
+
+       curl -v -u admin:geoserver -XPOST -H "Content-type: text/xml" 
+         -d "<wmtsStore><name>basemap-nat-geo-datastore</name><description>esri-street-map</description><capabilitiesURL>https://services.arcgisonline.com/arcgis/rest/services/NatGeo_World_Map/MapServer/WMTS/1.0.0/WMTSCapabilities.xml</capabilitiesURL><type>WMTS</type></wmtsStore>" 
+         http://localhost:8080/geoserver/rest/workspaces/acme/wmtsstores
+
+.. admonition:: python
```

Comment:
Thank you for the updates. I suggest removing the python and java sections, they might stay TBD for a very long time.